dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -123,6 +123,9 @@ http.messageSizeLimit.desc= Limits the acceptable size of an incoming message. I
 http.incomingBodyBufferSize=Incoming body buffer size
 http.incomingBodyBufferSize.desc=Specifies the size of each buffer used when reading the body of an incoming HTTP message.
 
+http.throwIOEForInboundConnections=Throw ioexception for inbound connections
+http.throwIOEForInboundConnections.desc=Specifies if the HTTP channel should throw an IOException when an inbound connection is closed while still in use by the servlet. The default value is set according to the configured servlet feature, being set to false prior to Servlet 4.0. As of Servlet 4.0, this option is true by default.
```

Comment:
change "Specifies if the HTTP channel should throw an IOException" to "Specifies whether the HTTP channel creates an I/O exception"

Change:
The default value is set according to the configured servlet feature, being set to false prior to Servlet 4.0. As of Servlet 4.0, this option is true by default.
to:
The default value is set according to the configured servlet feature.  Prior to Servlet 4.0, the default value is false; starting with Servlet 4.0, the default value is true.

If you can highlight (e.g., bold) the words false and true, it might improve readability.

Reply:
Change made